lib/spandex_datadog/adapter.ex

```diff
@@ -95,29 +174,26 @@ defmodule SpandexDatadog.Adapter do
     conn
     |> Plug.Conn.get_req_header(header_name)
     |> List.first()
-    |> parse_header()
   end
 
   @spec get_header(%{}, String.t()) :: integer() | nil
   defp get_header(headers, key) when is_map(headers) do
```

Comment:
Could the `get_header/2` function just have an additional clause for `Plug.Conn`? That could eliminate the code duplication above.